src/diffusers/loaders.py

```diff
@@ -1268,6 +1290,7 @@ def lora_state_dict(
             state_dict = pretrained_model_name_or_path_or_dict
 
         network_alphas = None
+        # TODO: replace it with `state_dict_utils` logic?
```

Comment:
A emthod from `state_dict_utils` you mean?

Reply:
Yes exactly